internal/pkg/agent/vault/vault_file_notwindows.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 const (
-	saltSize = 8
+	iterations = 12022
+	keyLen     = 32
```

Comment:
I think we should consider updating these in FIPS mode as we already use different values in `internal/pkg/crypto/hash.go` (higher iterations count, and longer keyLen)